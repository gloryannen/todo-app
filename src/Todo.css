.TodoContainer {
  display: flex;
}

.todoRemoveBtn {
  cursor: pointer;
  border: none;
  background-color: transparent;
  color: rgb(194, 4, 4);
  font-weight: bold;
  background-color: white;
}

li {
  background-color: rgb(251, 251, 251);
  border-bottom: 1px solid rgba(71, 71, 71, 0.157);
  border-right: 1px solid rgba(71, 71, 71, 0.157);
  padding: 3px;
  width: 100%;
  word-wrap: break-word;
  text-align: left;
  padding-left: 10px;
}
