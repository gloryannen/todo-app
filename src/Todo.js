import React from 'react';
import './Todo.css';

const Todo = ({ todo, id, handleRemoveTodo }) => {
  const removeTodo = () => handleRemoveTodo(id);

  return (
    <div className='TodoContainer'>
      <li>{ todo }</li>
      <button className='todoRemoveBtn' onClick={ removeTodo }> X </button>
    </div>
  )
}

export default Todo;