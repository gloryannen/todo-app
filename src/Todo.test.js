import React from 'react';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Todo from './Todo';

// Smoke test
it("renders without crashing", function () {
  render(<Todo />);
});

// Snapshot test
it("matches snapshot", function () {
  const { asFragment } = render(< Todo />);
  expect(asFragment()).toMatchSnapshot();
});
