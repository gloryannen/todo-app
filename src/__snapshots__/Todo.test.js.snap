// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="TodoContainer"
  >
    <li />
    <button
      class="todoRemoveBtn"
    >
       X 
    </button>
  </div>
</DocumentFragment>
`;
